plugins/occt/CMakeLists.txt

```diff
@@ -17,7 +17,13 @@ if("${OpenCASCADE_VERSION}" VERSION_LESS "7.5.0")
   message(FATAL_ERROR "Plugin: OpenCASCADE: ${OpenCASCADE_VERSION} is not supported by F3D, please update your OpenCASCADE installation.")
 endif()
 
-if((NOT TARGET "TKSTEP") AND (NOT TARGET "TKIGES") AND (NOT TARGET "TKMesh"))
+if("${OpenCASCADE_VERSION}" VERSION_LESS "7.8.0")
```

Comment:
no need for quotes I believe `${OpenCASCADE_VERSION}`

Reply:
@thierry-FreeBSD please adress this, I will then merge :)

Reply:
> @thierry-FreeBSD please adress this, I will then merge :)

At line #16, there is:
`if("${OpenCASCADE_VERSION}" VERSION_LESS "7.5.0")`
and I did not change it.
Isn't it better to keep the same style?

Reply:
looks like it was a previous issue. Ok to keep as is.